lib/final_destination.rb

```diff
@@ -8,6 +8,8 @@
 
 # Determine the final endpoint for a Web URI, following redirects
 class FinalDestination
+  MAX_TIME = 10
+  MAX_SIZE = 1_048_576 # 1024 * 1024
```

Comment:
Sorry just one more thing, I am assuming we have the pretty low 1MB because the onebox should just be getting the metadata it needs from the response, not an actual full response body? We can always raise/lower this later just curious about the size.